packages/node_modules/@webex/internal-plugin-conversation/src/decryption-transforms.js

```diff
@@ -42,6 +42,10 @@ export const transforms = toArray('inbound', {
     if (object.objectType === 'transcodedContent') {
       return Promise.all(object.files.items.map((item) => ctx.transform('decryptFile', key, item)));
     }
+    // use default encryption keyUrl for images instead of activity encryptionKeyUrl
```

Comment:
Is this comment correct? I thought `key` here was the default key and the one on the object is the specific one to the activity?

Reply:
Oh yeah, I would update comment.  I should comment something ``using object encryption keyUrl for images instead of activity encryptionKeyUrl``